tendrl/commons/jobs/__init__.py

```diff
@@ -110,28 +112,31 @@ def process_job(jid):
                 # "Timed out"
                 try:
                     job = job.load()
-                    job.status = "failed"
-                    job.save()
+                    if job.status == "new":
+                        job.status = "failed"
+                        job.save()
                 except etcd.EtcdCompareFailed:
                     pass
                 else:
                     job = NS.tendrl.objects.Job(job_id=jid).load()
                     _msg = str("Timed-out (>10min as 'new')")
                     job.errors = _msg
-                    job.save()
-                    if job.payload.get('parent') is None:
-                        alert_utils.alert_job_status(
-                            "failed",
-                            "Job timed out (job_id: %s)" % jid,
-                            integration_id=NS.tendrl_context.integration_id or
-                            job.payload['parameters'].get(
-                                'TendrlContext.integration_id'
-                            ),
-                            cluster_name=NS.tendrl_context.cluster_name or
-                            job.payload['parameters'].get(
-                                'TendrlContext.cluster_name'
+                    if job.status == "new":
```

Comment:
please add L122 and L123  inside the if block

Reply:
ack